docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+
+
+## Build
+
+To run UID2 Operator on a GCP Confidential Space enclave, download the official Docker image from the following Google Container Registry location:
+
+- [ghcr.io/iabtechlab/uid2-operator](ghcr.io/iabtechlab/uid2-operator) (**GWH_YS05 this redirects to https://github.com/iabtechlab/uid2-operator/pkgs/container/uid2-operator. Should I use that?**)
+
+For example:
+
+```
+docker pull ghcr.io/iabtechlab/uid2-operator
+```
+
+You can use the following command to build a non-certified UID2 operator container image from the source code:
+
+```
+# From the root source folder
+# Update project version in pom to "1.0.0-SNAPSHOT"
+
+mvn -B package -P gcp 
+cp -r target scripts/gcp-oidc/
+docker build ./scripts/gcp-oidc/. -t ghcr.io/iabtechlab/uid2-operator:v1.0.0-SNAPSHOT
+```
+## Deploy
+
+(**GWH_YS06 is there something re can say re when they deploy integ versus production? Do they do integ then production? Do they generally have both, or transition one to the other? Some general info would be helpful.**)
+
+The following environments are available:
+
+- Integration environment: see [Integration Deployment](#integration-deployment)
+- Production environment: see [Production Deployment](#production-deployment)
+
+### Integration Deployment
+
+To deploy a new UID2 Operator in the GCP Confidential Space Enclave into the integration environment, follow these steps.
+
+#### (For uid2 admin) Register Enclave ID in the Admin Portal
+
+1. Log in to the Admin Portal.
+
+2.  Go to the [GCP Enclave ID page](https://admin-integ.uidapi.com/adm/enclave-gcp-v2.html) and provide the following values to generate the Enclave ID: (**GWH_YS07 I cannot access this URL. Sure it's valid?**)
+    - The full digest for the image, with or without "sha256:".(**GWH_YS08 do we need the quotes and the colon on sha256?**)
+    - Environment: `Production` or `Integration`.
+    - Debug mode: `True` or `False`. (**GWH_YS09 Do they have to click any buttons or anything to generate the ID? (I can't get to this page)**)
+    
+      NOTE: Debug mode is only available in the integration environment. In Production, `False` is the only valid value.
+
+3. Save the `GCP Enclave ID` for use in the next step.
+
+2. Go to the [Enclave ID Management page](https://admin-integ.uidapi.com/adm/enclave-id.html) and provide the following values to register the GCP Enclave ID that you just generated:
+    - Name: enclave name
+    - Protocol: `gcp-oidc`
+    - Enclave ID: the `GCP Enclave ID` value that was generated in Step 2.
+
+(**GWH_YS10 I guess we don't need the above instructions but if so I need to clarify. For integration deployment does the partner ONLY do the below? Or do we have to do the above first, and if so, how does the partner know when we've done it? Not sure how it all fits together for the onboarding process.**)
```

Comment:
Above steps in (For uid2 admin) section will be handled by us ( uid2-admin ). 

As a partner they don't need to know those details.